solr/solrj/src/java/org/apache/solr/client/solrj/impl/CloudHttp2SolrClient.java

```diff
@@ -28,6 +28,7 @@
 import org.apache.solr.client.solrj.request.RequestWriter;
 import org.apache.solr.client.solrj.request.UpdateRequest;
 import org.apache.solr.common.SolrException;
+import org.apache.solr.common.cloud.SolrClassLoader;
```

Comment:
Do we need any additional testing around this on the `CloudHttp2SolrClient`, `CloudLegacySolrClient`?   You suggested they are covered already, just wanted to confirm.